fastapi-stock/main.py

```diff
@@ -46,6 +49,10 @@ class Price(BaseModel):
 )
 
 
+@app.get("/")
+async def docs():
+    return RedirectResponse("docs")
+
```

Comment:
I think connect already handles this implicitly--do we know what effect adding this snippet has there?

Reply:
This ensures the endpoint resolves in Workbench.